### DISCUSSION TITLE
How do you parse editor JSON to renderHTML format?

### DISCUSSION THREAD

**Discussion by vitovh** [ueberdosis/tiptap#1290] [General Questions] [answered]
Hi there! I am trying to make an extension where I am supposed to render a part of a document inside another document, I am very close just one last thing...
Is there a way to convert my JSON content to the format in the screenshot?
Example of the JSON content:
{"type":"doc","content":[{"type":"heading","content":[{"type":"text","text":"Hallo"}],"attrs":{"textAlign":"left","level":1}}]}
![screenshot](https://user-images.githubusercontent.com/59627539/117468742-4767d480-af55-11eb-92b3-7f41087db97e.png)

**Comment by appinteractive** [comment]
`editor.toHtml()` https://www.tiptap.dev/api/utilities/html/#generate-html-from-json

**Comment by hanspagel** [comment, marked as answer]
I think you’d need to look at Node views:
https://www.tiptap.dev/guide/node-views/js#render-a-node-view-with-javascript
